test/test_lyrics.py

```diff
@@ -457,28 +432,34 @@ def setUp(self):
             self.skipTest("Python's built-in HTML parser is not good enough")
 
 
-class LyricsGeniusScrapeTest(LyricsGeniusBaseTest):
-
-    """Checks that Genius backend works as intended.
-    """
-    import requests
+class GeniusScrapeLyricsFromSongPageTest(GeniusBaseTest):
+    """tests Genius.scrape_lyrics_from_song_page()"""
 
     def setUp(self):
         """Set up configuration"""
-        LyricsGeniusBaseTest.setUp(self)
+        GeniusBaseTest.setUp(self)
```

Comment:
Lots of classes prefacing with 'Lyrics' even though we are in the lyrics.py namespace. I removed it, but is that common convention for this repo?

Reply:
No, it's not a convention or anything. My best guess is that one of the initial tests did that, which sort of made sense in isolation, and then subsequent ones just copied it.